src/data/json-update-builder.js

```diff
@@ -0,0 +1,86 @@
+const OPERATIONS = {
+  SET         : 'JSON_SET',
+  INSERT      : 'JSON_INSERT',
+  REPLACE     : 'JSON_REPLACE',
+  REMOVE      : 'JSON_REMOVE',
+  ARRAY_APPEND: 'JSON_ARRAY_APPEND',
+  ARRAY_INSERT: 'JSON_ARRAY_INSERT'
+}
+
+const OPERATION_FIELD_NAME = '___operation'
+const ARGS_FIELD_NAME = 'args'
+
+export default class JSONUpdateBuilder {
+  constructor() {
```

Comment:
can we add `operationName` and `args` as arguments here in the constructor ?

Reply:
I thought that with zero arguments in the constructor we will force users to use our methods rather than just passing args to the constructor because IDE will suggest this way

Reply:
developers should use it as it described in the docs